### PR TITLE
Refactor: return null instead of undefined

### PR DESCRIPTION
**Pull request by Fcmam5**
# Description

A proposition to move from returning `undefined` to `null`. For more information please follow [our thread on slack](https://dzcode.slack.com/archives/C01A60X834N/p1602326905004800).

This PR adds a JEST rule forcing 100% test coverage.

# Checklist:

- [x] I checked that there's no dataset update (can be done by running `npm run update-dataset`)
- [x] `npm test` passes on my machine
- [x] `npm run lint` passes on my machine

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dzcode-io/leblad",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (ZibanPirate) [inline package.json:3]: Shouldn't this be a breaking change?

```suggestion
  "version": "1.0.0",
```

**Review by ZibanPirate — APPROVED** [review]
All good, just one concern about the type of change here, feel free to resolve it if u think it's not an issue

File: package.json (hunk 1 above)
Comment (Fcmam5) [reply package.json:3]: I bumped the patch version as the library is not that mature, so I thought of releasing v1.0 with all our new changes.
What do you think?

Comment (ZibanPirate) [reply package.json:3]: In that case you could've gone with '.alpha.xx' but since no one is using it for now :') it's fine 👌🏽

**Pull request merged**